common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -218,6 +218,17 @@ class InheritanceMixin(XBlockMixin):
         default=False
     )
 
+    self_paced = Boolean(
+        display_name=_('Self Paced'),
+        help=_(
+            'Set this to "true" to mark this course as self-paced. Self-paced courses do not have '
+            'due dates for assignments, and students can progress through the course at any rate before '
```

Comment:
`Self-paced courses do not have due dates for assignments. Students can progress through the course at their own pace`